### DISCUSSION TITLE
关于上线会员功能后的流量限制猜想

### DISCUSSION THREAD

**Discussion by raydoom** [messense/aliyundrive-webdav#348] [General]
今天在使用webdav功能下载的时候，突然日志报错（429 Too Many Requests）
随后使用了阿里云官方客户端，下载时显示下载失败，小白羊最新版，一直显示下载启动中
在网上看到有人说时超出了流量限制了，以下是[GJG666](https://github.com/GJG666)测试的结果：
第一次限制是24小时后解除，第二次限制需要一周才能自动解除。所以如果你是第一次被限制，那这个"当日"，确实是第二天就可以下载了，但如果你是第二次被限制，那这个"当日"可能会连续持续七天。开通会员可以秒解除下载限制，普会和超会都可以

为了验证，我也开了个普通会员，然后，可以正常下载了
准备把文件下载下来，以后不用了

**Comment by messense** [comment]
白嫖是不可持续的，至于阿里云盘功能特性是否值得付费用脚投票呗。

**Comment by raydoom** [comment]
人家确实有成本，不过暗地里做限制，明着说不限速，这就有点迷了

**Reply by messense** [reply]
确实不限速，但是限制流量😂

**Comment by le0zh0u** [comment]
我开了会员，还是抛出一样的错误 请求失败，url=/file/get_down
load_url, code=429, body={"code":"TooManyDownloadRequests","message":"TooManyDownloadRequests","requestId":"0bc13b0f164923507
36487857e2807","resultCode":"TooManyDownloadRequests"}                                            
裂开了。需要重新登录啥的吗
